backend/handlers/handlers.go

```diff
@@ -1370,7 +1370,7 @@ func (handlers *Handlers) getBtcDirectSupported(r *http.Request) interface{} {
 	regionCode := r.URL.Query().Get("region")
 	return Result{
 		Success:   true,
-		Supported: exchanges.IsBtcDirectSupported(acct.Coin().Code(), regionCode),
+		Supported: exchanges.IsBtcDirectSupported(acct.Coin().Code()) && exchanges.IsRegionSupportedBtcDirect(regionCode),
```

Comment:
I think that we should refactor a bit this endpoint:
- rename to getExchangeBtcDirectOTCSupported (don't know why it wasn't already like that)
- maybe move the logic inside a dedicated method exchanges.IsBtcDirectOTCSupported ?

Reply:
Also, could you add `IsBtcDirectSupported` check inside `getExchangeSupported` (and maybe move that logic inside exchanges too)? :pray: 

Otherwise we won't consider btcDirect when loading the accounts list in the buy&sell page